packages/mold.rb

```diff
@@ -46,5 +46,10 @@ def self.build
 
   def self.install
     system "DESTDIR=#{CREW_DEST_DIR} samu -C builddir install"
+    # See https://github.com/rui314/mold/issues/697
+    return unless ARCH == 'x86_64'
+
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/lib/mold"
+    FileUtils.ln "#{CREW_DEST_LIB_PREFIX}/mold/mold-wrapper.so", "#{CREW_DEST_PREFIX}/lib/mold/mold-wrapper.so"
```

Comment:
`FileUtils.ln "#{CREW_LIB_PREFIX}/mold/mold-wrapper.so", "#{CREW_DEST_PREFIX}/lib/mold/mold-wrapper.so"`?

Reply:
No , as this is making a hard link, not a symlink, so you need the actual path to find the file.

Is that bad?  I think our system handles hard links just fine...

Reply:
Will the hard link point to `/usr/local/tmp/crew/dest/...` after installation? If so using `cp` or symlink might be a better choice

Reply:
What benefit is there to a hardlink vs a symlink?